src/main/java/com/example/SqlInterceptorImpl.java

```diff
@@ -0,0 +1,20 @@
+package com.example;
+
+import io.prometheus.client.Counter;
+
+// TODO: Not threadsafe
+public class SqlInterceptorImpl implements SqlInterceptor {
+
+  private static final Counter sqlCount = Counter.build().name("sql_count").help("help me").register();
+
+  @Override
+  public void start() {
+    System.out.println("In end");
```

Comment:
Ah, we mixed up the start/end messages 😆

Reply:
that makes more sense lmao